spec/features/invoices/index_spec.rb

```diff
@@ -35,4 +35,42 @@
       expect(subject).to_not have_content(org2_invoice.customer)
     end
   end
+
+  context 'complete invoice', js: true do
+    let!(:account)  { create :bank_account, organization: org }
+    let!(:category) { create :category, :income, organization: org }
+    let!(:invoice)  { create :invoice, organization: org, amount: 500 }
+    let(:comission) { Money.new(100, invoice.currency) }
+
+    before do
+      visit invoices_path
+      click_on 'Complete Invoice'
+      within '#new_transaction' do
```

Comment:
@yunixon add extra case on this page, with checking form. Make sure in spec that form shows only Income categories, only Accounts with Invoice.currency. Show Amount and Commission fields.